### PR TITLE
[Tech Debt] Update readme.md

### PR DESCRIPTION
**Pull request by kcw-grunt**

### CODE REVIEW

File: pages/development.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+4. Once the build is approved, a PR to merge the `release/v2.9.0` into `master` is pushed to Github.
+
+5. Once approved, it's merged.
+   
+
+
+
+#### Step B
+6. A new PR from the `master` to `develop` is created
```
Comment (mosadialiou) [inline pages/development.md:23]: should be: `

6. A new PR from the `release/v2.9.0` to `develop` is created`

**Review by mosadialiou — CHANGES_REQUESTED** [review]

**Review by mosadialiou — CHANGES_REQUESTED** [review]
LGTM 
However the destination branch of the PR has to be `develop`

**Comment by kcw-grunt** [comment]
> LGTM
> However the destination branch of the PR has to be `develop`

Cool.  Changed.  Tx.

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**